### DISCUSSION TITLE
Ventoy update from 1.0.55 to 1.0.64 hangs at 22%

### DISCUSSION THREAD

**Discussion by hundeleine** [ventoy/Ventoy#1413] [General]
Ventoy (**32 & x64**) hangs everytime at 22% if update on device USB-flash-memory (64GB , MBR) from ver. 1.0.**55** to ver. 1.0.64.
Update an USB device from ver. 1.0.62 to ver. 1.0.64 works fine.

Search in log.txt for "**Intenso Ultra Line 64GB**" logical drive "**O:**" .
[log.txt](https://github.com/ventoy/Ventoy/files/7968020/log.txt)

**Comment by ventoy** [comment]
You can put the Ventoy-1.0.64-LiveCD.iso in Ventoy USB and reboot your computer.
When you boot into Ventoy menu, select the VentoyLiveCD.iso to boot.
After boot you will run into Linux version of Ventoy GUI and update your Ventoy here.

**Reply by summerlotus513** [reply]
I also encountered the same problem. The latest version 1.0.73 version, I upgrade from 1.0.64, will be 22%
